consul_exporter.go

```diff
@@ -474,6 +476,7 @@ func main() {
 	kingpin.Flag("consul.timeout", "Timeout on HTTP requests to the Consul API.").Default("500ms").DurationVar(&opts.timeout)
 	kingpin.Flag("consul.insecure", "Disable TLS host verification.").Default("false").BoolVar(&opts.insecure)
 	kingpin.Flag("consul.request-limit", "Limit the maximum number of concurrent requests to consul, 0 means no limit.").Default("0").IntVar(&opts.requestLimit)
+	kingpin.Flag("consul.max-connections", "Limit the maximum number of concurrent connections to consul, 0 means no limit.").Default("0").IntVar(&opts.maxConns)
```

Comment:
Maybe we should set this to a sane value? The suggested 16 seems reasonable to me.

Reply:
Yup, can change that. I think 16 would be large enough for most installations to work fine (8 also seems to work for us) and low enough to prevent overloading consul.